src/test/resources/BulkCopyCSVSendTemporalDataTypesAsStringForBulkCopy2.csv

```diff
@@ -0,0 +1,2 @@
+int,date,datetime,datetime2,time,datetimeoffset,smalldatetime,money,smallmoney

```

Comment:
Why do you need separate file for this test?

Reply:
I wanted to test the cases where the data in CSV doesn't "exactly" match the returned data. For example, it's legal to omit the nanosecond part of datetime2 value in the CSV, but the result from the server will have `0000000` (7 digits of nanoseconds) attached to the data. I wanted a separate CSV file to test those cases easily. It's only used in the `testSendTemporalDataTypesAsStringForBulkCopyOptional` method in my test class.